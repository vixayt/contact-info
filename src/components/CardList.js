import React from 'react';
import Card from './Card';

const CardList = ({ users }) => {
  console.log('u', users )
  return (
    <div>
      {users.map((user, i) => {
        return (
          <Card 
            key={users[i].id}
            id={users[i].id} 
            name={users[i].name} 
            username={users[i].username}
            email={users[i].email}
          />
          )
        })
      }
    </div>
  );
}
export default CardList;