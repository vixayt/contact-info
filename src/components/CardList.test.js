import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('should render CardList component', () => {
    const mockUsers = [
        {
            id: 1,
            name: "John Snow",
            username: "John",
            email: "kingofthenorth@gmail.com"
        }
    ]
    expect(shallow(<CardList users={mockUsers} />)).toMatchSnapshot();
})
