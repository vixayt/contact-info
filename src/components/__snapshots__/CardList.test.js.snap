// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CardList component 1`] = `ShallowWrapper {}`;
